djangocms_content_expiry/cms_config.py

```diff
@@ -147,6 +147,31 @@ def content_expiry_site_alias_excluded_set(queryset, request):
     )
 
 
+def get_copy_compliance_number_button(obj):
+    """
+    Return a user friendly link to copy a content expiry compliance number to other Moderation Request Items
+    link redirects to view which handles this
+    """
+    version = obj.moderation_request.version
+
+    if hasattr(version, "contentexpiry"):
+        content_expiry = version.contentexpiry
+        view_endpoint = format_html(
```

Comment:
Not a blocker as it works as is, but it looks like this is building up a string rather than html content, so could be done with the string `format()` method rather than `format_html`?

Also as an aside, when I build up a url with a querystring I like to use [urlencode](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.http.urlencode):
```
In [1]: from django.utils.http import urlencode

In [2]: querystring = urlencode({
   ...:     "foo": 1,
   ...:     "bar": 2,
   ...: })

In [3]: url = f"/foobar/?{querystring}"

In [4]: url
Out[4]: '/foobar/?foo=1&bar=2'
```